src/subscript/merge_rft_ertobs/merge_rft_ertobs.py

```diff
@@ -101,9 +154,10 @@ def get_observations(obsdir="", filepattern="*.obs"):
     for obsfilename in glob.glob(os.path.join(obsdir, filepattern)):
         # Warning: Deducing the wellname this way will fail for
         # exotic filepatterns.
-        wellname = os.path.basename(obsfilename).split(filepattern.replace("*", ""))[
-            0:-1
-        ][0]
+        wellname_reportstep = os.path.basename(obsfilename).split(
+            filepattern.replace("*", "")
```

Comment:
Isn't the `*` already gone due to `obsfilename` being a result from `glob.glob`?

Reply:
It is gone in `obsfilename`, but here we remove it from the filepattern in order to chop away the extension. I see `split()` could have been replace by `replace` also, but the way the code works here is suboptimal in any case (relying on filenames).